controller/system_rollout_controller.go

```diff
@@ -1719,6 +1721,9 @@ func (c *SystemRolloutController) restoreVolumes() (err error) {
 
 				reason := fmt.Sprintf(constant.EventReasonFailedCreatingFmt, types.LonghornKindVolume, restore.Name)
 				c.eventRecorder.Event(c.systemRestore, corev1.EventTypeWarning, reason, message)
+
+				c.ignorePersistenVolumeDueToMissingVolume(&restore)
+				c.ignorePersistenVolumeClaimDueToMissingVolume(&restore)
```

Comment:
Why not just apply the skipped labels to those skipped resources, then they can be bypassed during restoring PV/PVC?

```go
		annos[types.GetLastSkippedSystemRestoreLabelKey()] = systemRestoredURL
		annos[types.GetLastSkippedSystemRestoreAtLabelKey()] = systemRestoredAt
```

Reply:
Not sure if that is necessary.
1. The PV and PVC will not get created so the skipped annotation will not be seen by the users.
1. There will be 2 more arguments(systemRestoredURL, systemRestoredAt) to pass through to `ignorePersistenVolume*`  functions.
1. The backup PV and PVC resources might already have the annotations so we need to check to see if those are from the current rollout. Or, we need to remove them during the unpacking.

Reply:
After discussing with @innobead , we can still apply the skipped annotations to use as filtering for restoring resources.